cli/popper/commands/cmd_dot.py

```diff
@@ -49,6 +47,16 @@ def add_to_graph(graph_str, wf, parent, children):
     for wfile in wfile_list:
         pipeline = WorkflowRunner(wfile, False, False, False, False, True)
         wf = pipeline.wf
-        workflow_name = wf.name.replace(' ', '_').replace('-', '_')
-        graph_str = add_to_graph("", wf, workflow_name, wf.root)
-        log.info("digraph G {\n" + graph_str + "}\n")
+        workflow_name = wf.name
+        graph_set = add_to_graph(set(), wf, workflow_name, wf.root)
+        graph_str = '\n'.join(graph_set)
+        workflow_attr = '[ shape=diamond, bordercolor=blue, border=bold]'
```

Comment:
Can we make this a rounded (transparent) box? Something like:

```
[shape="box", style="filled,rounded", color="crimson", fillcolor="transparent"]
```

this with the goal of having the same symbols (and meaning) as in traditional flowcharts (rounded box denoting the start of a process, and boxes denoting subprocesses). The use of `cyan` would make the diagram amenable to both dark and light terminal backgrounds (assuming something like [`imgcat`](https://github.com/eddieantonio/imgcat#build) is being used)